src/test/resources/com/scriptbasic/testprograms/TestSelectBadSyntax4.bas

```diff
@@ -0,0 +1,9 @@
+'
+' This program is used to test the bad syntax of 
+' command select case ... end select
+'
```

Comment:
may seem obvious when you create, but I feel like this needs another comment line:

`' when the 'end select' is missing`